src/autora/experiment_runner/synthetic/psychology/stroop_model.py

```diff
@@ -0,0 +1,267 @@
+        ],
+        dependent_variables=[response_green],
+    )
+
+    rng = np.random.default_rng(random_state)
+
+    class StroopModel:
+        def __init__(self, choice_temperature, std=0.):
+            self.choice_temperature = choice_temperature
+            self.std = std
+    
+            # define affine transformations
+            self.input_color_hidden_color = self.init_linear(2, 2)
+            self.input_word_hidden_word = self.init_linear(2, 2)
+            self.hidden_color_output = self.init_linear(2, 2)
+            self.hidden_word_output = self.init_linear(2, 2)
+            self.task_hidden_color = self.init_linear(2, 2)
+            self.task_hidden_word = self.init_linear(2, 2)
+    
+            self.bias = -4
+    
+            self.init_weights()
+
+        def init_linear(self, in_features, out_features):
+            return rng.random.rand(out_features, in_features) * 0.01
+    
+        def init_weights(self):
+            self.input_color_hidden_color = self.init_linear(2, 2)
+            self.hidden_color_output = self.init_linear(2, 2)
+            self.input_word_hidden_word = self.init_linear(2, 2)
+            self.hidden_word_output = self.init_linear(2, 2)
+            self.task_hidden_color = self.init_linear(2, 2)
+            self.task_hidden_word = self.init_linear(2, 2)
+    
+        def sigmoid(self, x):
+            return 1 / (1 + np.exp(-x))
+        
+
+        def forward(self, input):
+            input = np.array(input)
+            if len(input.shape) <= 1:
+                input = input.reshape(1, len(input))
+    
+            color = np.zeros((input.shape[0], 2))
+            word = np.zeros((input.shape[0], 2))
+            task = np.zeros((input.shape[0], 2))
+    
+            color[:, 0:2] = input[:, 0:2]
+            word[:, 0:2] = input[:, 2:4]
+            task[:, 0:2] = input[:, 4:6]
+    
+            color_hidden = self.sigmoid(
+                np.dot(self.input_color_hidden_color, color.T)
+                + np.dot(self.task_hidden_color, task.T)
+                + self.bias
+            ).T
+    
+            word_hidden = self.sigmoid(
+                np.dot(self.input_word_hidden_word, word.T)
+                + np.dot(self.task_hidden_word, task.T)
+                + self.bias
+            ).T
+    
+            output = np.dot(self.hidden_color_output, color_hidden.T) + np.dot(
+                self.hidden_word_output, word_hidden.T
+            )
+    
+            if self.std > 0:
+                output += rng.random.randn(*output.shape) * self.std
+    
+            output_softmaxed = np.exp(output * 1 / self.choice_temperature) / (
+                np.exp(output[0] * 1 / self.choice_temperature)
+                + np.exp(output[1] * 1 / self.choice_temperature)
+            )
+
+            return output_softmaxed
+
+    def experiment_runner(
+        conditions: Union[pd.DataFrame, np.ndarray, np.recarray],
+        observation_noise: float = 0.01,
```

Comment:
rename to added_noise
```suggestion
        added_noise: float = 0.01,
```